include/exadg/operators/operator_base.h

```diff
@@ -229,10 +243,19 @@ class OperatorBase : public dealii::Subscriptor
    * rhs only make sense for linear operators (but they have e.g. no meaning for linearized
    * operators of nonlinear problems). For this reason, these functions are currently defined
    * 'virtual' to provide the opportunity to override and assert these functions in derived classes.
+   *
+   * For continuous Galerkin discretizations, this function calls internally the member function
```

Comment:
Sorry for nit-picking, but isn't it more grammatically correct to use
```suggestion
   * For continuous Galerkin discretizations, this function internally calls the member function
```